### DISCUSSION TITLE
How to save and load model resulted from AutoML ?

### DISCUSSION THREAD

**Discussion by kusumy** [alan-turing-institute/sktime#5234] [General]
On the user guide "Forecasting in sktime", there is section to perform AutoML (ex, using MultiplexForecaster), and let say i add reduction based forecaster (ex, Random Forest) to the model list. My question is how to save the model resulted from AutoML, and loaded for later use?

Regards

**Comment by fkiraly** [comment]
For saving, you can use `my_fitted_model.save()` or `save(file_location)`, this will save the model as a pickle or similar format (for deep learning) to a memory or file location.

For loading, use `from sktime.base import load` on the file, or in-memory container.

**Comment by FuchtelJockel** [comment]
But this doesn't seem to work with classifiers that contain multiple models like the BaggingClassifier. After training with `fit()` I get a very tiny zip file without the actual models contained.

I can iterate over the `estimators_` attribute and call `save()` on each one, but how do I rebuild everything after that?

Similar story for the `RocketClassifier`.
